package/Dockerfile

```diff
@@ -1,14 +1,13 @@
 FROM nginx:1.14.0-alpine
 
-RUN apk -U --no-cache add bash curl wget git ca-certificates openssl \
+RUN apk -U --no-cache add bash curl wget git ca-certificates openssl openssh\
     && mkdir -p /opt/rke-tools/bin /etc/confd \
     && wget https://storage.googleapis.com/kubernetes-release/release/v1.11.2/bin/linux/amd64/kubectl -O /usr/local/bin/kubectl \
     && wget https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.16.0-linux-amd64 -O /usr/local/bin/confd \
-    && chmod +x /usr/local/bin/confd /usr/local/bin/kubectl \
-    && apk --no-cache del curl
-
+    # FIXME wait for new release of rancher cli for better scripting
```

Comment:
we should never LGTM till all the fixme are addressed :) should have checked before merging. Please address in the follow up pr